modified:   app/controllers/api/v1/immunization_report_controller.rb

dominickasanga opened the pull request:
range pickers for weeks and months

dominickasanga committed:
modified:   app/controllers/api/v1/immunization_report_controller.rb
modified:   config/routes.rb
new file:   vendor/cache/et-orbi-1.2.11.gem
new file:   vendor/cache/fugit-1.11.0.gem
new file:   vendor/cache/raabro-1.4.0.gem
new file:   vendor/cache/sidekiq-cron-1.12.0.gem
deleted:    vendor/cache/sidekiq-unique-jobs-8.0.10.gem

dominickasanga committed:
modified:   app/controllers/api/v1/immunization_report_controller.rb

dominickasanga committed:
Merge branch 'chore/find_patients_pagination' of https://github.com/HISMalawi/BHT-EMR-API into date_picker

dominickasanga committed:
Revert "Merge branch 'chore/find_patients_pagination' of https://github.com/HISMalawi/BHT-EMR-API into date_picker"

This reverts commit 7f61aae5e87a56beff7005e433dd1f683da4b193, reversing
changes made to c7e44fe47916122a327d9f1c77c1ef31d241c8c6.

dominickasanga committed:
modified:   app/controllers/api/v1/immunization_report_controller.rb
new file:   app/services/immunization_service/reports/general/vaccines_administered.rb
modified:   config/routes.rb

dominickasanga committed:
modified:   app/controllers/api/v1/immunization_report_controller.rb
modified:   app/controllers/api/v1/pharmacy/items_controller.rb
modified:   app/services/immunization_service/reports/general/vaccines_administered.rb

dominickasanga committed:
new file:   app/models/pharmacy_batch_vvm.rb

dominickasanga committed:
modified:   app/services/name_search_service.rb

dominickasanga committed:
modified:   app/services/person_service.rb

dominickasanga committed:
modified:   app/services/stock_management_service.rb

dominickasanga committed:
new file:   db/migrate/20240812143220_add_details_to_pharmacy_batch_items.rb

dominickasanga committed:
new file:   db/migrate/20240813033538_create_pharmacy_batch_vvms.rb

dominickasanga committed:
new file:   vendor/cache/sidekiq-unique-jobs-8.0.10.gem

dominickasanga committed:
deleted:    vendor/cache/sidekiq-unique-jobs-8.0.10.gem

dominickasanga committed:
Merge branch 'chore/find_patients_pagination' of https://github.com/HISMalawi/BHT-EMR-API into date_picker

fchiyenda commented on app/controllers/api/v1/immunization_report_controller.rb:
```diff
@@ -10,6 +10,86 @@ def stats
+    end
+
+    def vaccines_administered
+        start_date = params[:start_date]
```
Would recommend to permit params and require them as well

dominickasanga committed:
modified:   app/controllers/api/v1/immunization_report_controller.rb

dominickasanga committed:
modified:   app/services/immunization_service/reports/general/vaccines_administered.rb

fchiyenda committed:
Required params

dominickasanga committed:
modified:   app/services/immunization_service/reports/general/vaccines_administered.rb

dominickasanga committed:
modified:   app/services/immunization_service/reports/general/vaccines_administered.rb

dominickasanga committed:
modified:   app/services/immunization_service/reports/general/vaccines_administered.rb

dominickasanga committed:
modified:   app/services/immunization_service/reports/general/vaccines_administered.rb

dominickasanga committed:
modified:   app/controllers/api/v1/immunization_report_controller.rb
modified:   config/routes.rb

dominickasanga committed:
modified:   app/controllers/api/v1/immunization_report_controller.rb
modified:   app/services/immunization_service/reports/general/vaccines_administered.rb

dominickasanga committed:
modified:   app/controllers/api/v1/immunization_report_controller.rb
modified:   app/services/immunization_service/reports/general/vaccines_administered.rb

dominickasanga committed:
modified:   app/controllers/api/v1/immunization_report_controller.rb

fchiyenda committed:
Merge branch 'date_picker' of https://github.com/HISMalawi/BHT-EMR-API into date_picker

fchiyenda committed:
Validate params before action

dominickasanga committed:
modified:   app/controllers/api/v1/immunization_report_controller.rb

dominickasanga committed:
modified:   app/services/immunization_service/reports/general/vaccines_administered.rb

dominickasanga committed:
modified:   app/services/immunization_service/reports/general/vaccines_administered.rb
